addons/analytic/static/src/js/analytic_distribution/analytic_distribution.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+    <t t-name="analytic_distribution" owl="1">
+        <div class="analytic_distribution" t-ref="analyticDistribution" t-on-keydown="onWidgetKeydown" t-att-title="''">
+
+            <t t-if="!props.readonly">
+                <div class="o_input o_input_dropdown" tabindex="0" t-ref="mainElement" t-on-focus="onMainElementFocus" t-on-click="() => this.resetRecentlyClosed()">
```

Comment:
There is an issue here - and it differs between form and list views:
when the popup is visible - press escape (the popup closes) - clicking on the `o_dropdown_button` does not re-open the popup.
Clicking below the tags works, but a click close to the tags does not.